optuna/storages/rdb/alembic/versions/v0.16.0.a_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
+
+Revision ID: v0.16.0.a
+Revises: v0.9.0.a
+Create Date: 2019-09-14 15:45:41.458947
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'v0.16.0.a'
```

Comment:
It might be better to update revision if the revision should be the same with a release version of Optuna.